server/build.go

```diff
@@ -145,6 +149,26 @@ func (task *BuildTask) Build() (esm *ESM, err error) {
 }
 
 func (task *BuildTask) build() (esm *ESM, err error) {
+	if task.Target == "raw" {
+		filePath := path.Join(task.wd, "node_modules", task.Pkg.Name, task.Pkg.Submodule)
+		log.Infof(filePath)
+
+		var file io.Reader
+		file, err = os.Open(filePath)
+		if err != nil {
+			return
+		}
+
+		saveFilePath := path.Join("raw", task.Pkg.String())
+
+		_, err = fs.WriteFile(saveFilePath, file)
```

Comment:
This is the copy file way to implement the static file server. 
The other way can just to install the package and serve the `${TMP_DIR}esm` directory directly.

Reply:
Wanted to get your thoughts on this.